tests/core/pm-module/packages/owned/1.0.0.json

```diff
@@ -0,0 +1,21 @@
+{
+  "manifest_version": "2",
+  "version": "1.0.0",
+  "package_name": "owned",
+  "meta": {
+    "license": "MIT",
+    "authors": [
+      "Piper Merriam <pipermerriam@gmail.com>"
+    ],
+    "description": "Reusable contracts which implement a privileged 'owner' model for authorization.",
+    "keywords": [
+      "authorization"
+    ],
+    "links": {
+      "documentation": "ipfs://QmUYcVzTfSwJoigggMxeo2g5STWAgJdisQsqcXHws7b1FW"
+    }
+  },
+  "sources": {
+    "./contracts/Owned.sol": "ipfs://Qme4otpS88NV8yQi8TfTP89EsQC5bko3F5N1yhRoi6cwGV"
```

Comment:
Should these be represented in their *compact* form.  I don't think I've seen a PR in ethpm to enforce that data format, so it's probably something that needs to have an issue created for, but I suspect we'll run into an error in this repository at some point in the future if we use non-canonical representations of the manifests here.

Reply:
Good point, I'll add that issue to `py-ethpm`. And once it gets implemented there it'll force us to update the contracts here.

Reply:
Should I understand "compact" to mean `ipfs:Qme...`?

Reply:
@davesque I don't think it has anything to do with the contents of the uri - just to enforce the guidelines [here in ethpm-spec](http://ethpm.github.io/ethpm-spec/package-spec.html#document-format)